scripts/cleanup-test-processes.sh

```diff
@@ -1,22 +1,34 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # This script can be handy to kill test processes launched manually
 # for debugging or still hanging around for other reasons.
 
-pkill --echo --uid "${UID}" --full '/monerod .* --regtest '
-pkill --echo --uid "${UID}" --full '/ganache.* --deterministic '
-pkill --echo --uid "${UID}" --full '/relayer'
+pkill_cmd=(pkill --echo --uid "${UID}" --full)
+if [[ "$(uname)" == 'Darwin' ]]; then
+	pkill_cmd=(pkill -l -U "${UID}" -f)
+fi
+
+echo "Stoping any monerod regest instances"
```

Comment:
There is no `--echo` flag equivalent for the macOS `pkill` command, so I added these echos. They are arguably helpful even on Linux.